rhtap-cli/rhtap-full-install.sh

```diff
@@ -5,60 +5,112 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+## The default catalog URL of the developer hub
+export DEVELOPER_HUB__CATALOG__URL="<REPLACE_ME, for example: https://github.com/redhat-appstudio/tssc-sample-templates/blob/main/all.yaml>"
 
-github_app_private_key_path="<REPLACE_ME>"
-
-export GITHUB__APP__ID="<REPLACE_ME>"
+## Existing Github app, see function "githubapp_integration"
 export GITHUB__APP__CLIENT__ID="<REPLACE_ME>"
 export GITHUB__APP__CLIENT__SECRET="<REPLACE_ME>"
-export GITHUB__APP__PRIVATE_KEY=$(cat $github_app_private_key_path)
-export GITHUB__APP__WEBHOOK__SECRET="<REPLACE_ME, a random string>"
+export GITHUB__APP__ID="<REPLACE_ME>"
+export GITHUB__APP__PRIVATE_KEY="<REPLACE_ME, base64 encoded key, for example: $(cat rhtap-test.2024-03-26.private-key.pem | base64 -w 0) >"
 export GITOPS__GIT_TOKEN="<REPLACE_ME>"
-export QUAY__DOCKERCONFIGJSON="<REPLACE_ME>"
-export QUAY__API_TOKEN="<REPLACE_ME>"
-export ACS__CENTRAL_ENDPOINT="<REPLACE_ME>"
-export ACS__API_TOKEN="<REPLACE_ME>"
+export GITHUB__APP__WEBHOOK__SECRET="<REPLACE_ME, a random string>"
 
-install_rhtap() {
-  # Path to your values.yaml.tpl file
-  tpl_file="charts/values.yaml.tpl"
+## Exisitng Gitlab server, see function "gitlab_integration"
+export GITLAB__TOKEN="<REPLACE_ME, create a Personal access tokens in page https://gitlab.com/-/user_settings/personal_access_tokens>"
+
+## External Jenkins server, see function "jenkins_integration", see the exiting Jenkins service in doc https://docs.google.com/document/d/1MZAFHUs2DHFNz1n11kKg9dqbfF9akbc_OMCD8mYbu8g/edit#heading=h.utupxh9dx34z
```

Comment:
Use link to this readme: https://github.com/xinredhat/rhtap-utils/blob/main/jenkins/README-JENKINS.md
We should not share on GitHub links to our private docs